capstone/deployment.py

```diff
@@ -101,3 +111,137 @@ def add_sys_path(path):
 
 def get_deployment_root() -> Path:
     return Path(config.data_dir) / "deployments"
+
+NOMAD_JOB_TEMPLATE = """
+job "{{name}}" {
+  type = "service"
+
+  group "{{name}}" {
+    count = 1
+
+    network {
+      port "web" {
+        to = 8080
+      }
+    }
+
+    service {
+      name     = "{{name}}"
+      port     = "web"
+
+      tags = ["capstone-service"]
+
+      meta {
+        host = "{{host}}"
+      }
+    }
+
+    task "{{name}}" {
+
+      driver = "docker"
+
+      config {
+        image = "{{docker_image}}"
+        force_pull = true
+        ports = ["web"]
+      }
```

Comment:
where do we specify the local docker registry to pull this image from?

Reply:
docker image name is generated. See `docker_registry` in `config.py`.